__tests__/index.test.js

```diff
@@ -28,6 +28,38 @@ describe("GET /api/topics", () => {
   });
 });
 
+describe("GET /api/articles", () => {
+  test("200 - retrieves array of objects representing articles in db and loops through checking properties", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        const { articles } = body;
+        expect(articles).toBeInstanceOf(Array);
+        expect(articles).toHaveLength(12);
+        articles.forEach((article) => {
+          expect(
+            typeof article.author &&
```

Comment:
This code isn't doing what you think it's doing i'm afraid!

you would need to write it like this

```js
expect(
    typeof article.author === 'string' &&
    typeof article.title === 'string' &&
    typeof article.topic === 'string' &&
    // ... etc    
    )
```

a neater way would be to use `expect.any(String)`

Reply:
okay, thanks for the help will work on the changes now